lib/skunk/cli/application.rb

```diff
@@ -36,6 +39,16 @@ def warn_coverage_info
         warn "warning: Couldn't find coverage info at #{COVERAGE_FILE}."
         warn "warning: Having no coverage metrics will make your SkunkScore worse."
       end
+
+      # :reek:NilCheck
+      def print(message)
+        filename = @parsed_options[:output_filename]
+        if filename.nil?
+          $stdout.puts(message)
+        else
+          File.open(filename, "w") { |file| file.puts(message) }
```

Comment:
@bronzdoc My only concern is that with this approach we're opening and closing immediately the file for each line we output.

Reply:
We are not writing line by line we just open an IO object and write a complete string to it. But maybe I'm missing something?

Reply:
🤔 I think you're right.